index.html

```diff
@@ -90,7 +90,15 @@
 
       <div class="row">
         <div class="span15 alert-message block-message">
-          <h2>祝！完売 (2012/01/13)</h2>
+          <h2>タイピング Professionals 電子版の無償公開スタート！(2020/12)</h2>
+          <p>タイピング Professionals 書籍版をご購入頂いた方々、誠にありがとうございました。<br />
+          第2刷の販売から早8年。執筆陣の情熱が詰まったこの1冊は、非常に入手困難な状況にあります。<br />
+          この度、当時買いそびれてしまった方や、新しくタイピングの世界に参戦された方のため、電子版（PDF）を無償公開させていただく運びとなりました！<br />
+          多彩な執筆陣による情熱のこもった記事の数々を、心ゆくまでお楽しみください。</p>
+          <h3><a href='gachityping_v2.pdf'><u>[タイピング Professionals（第2刷） 電子版]</u></a></h3>

```

Comment:
```suggestion
          <h3><a href='https://github.com/typing/book_typing_professionals/releases/download/not_v2/gachityping_v2.pdf'><u>[タイピング Professionals（第2刷） 電子版]</u></a></h3>
```
github の Releases という機能を使って PDF ファイル自体はアップロード済みなので、suggestion の通りそのファイルに対して直リン貼ってもらえれば大丈夫です！